converter.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import os
+import fnmatch
+import json
+from scanomatic.io import jsonizer
+from scanomatic.models.factories.compile_project_factory import (
+    CompileImageAnalysisFactory,
+    CompileProjectFactory,
+)
+from scanomatic.io.paths import Paths
+from scanomatic.io.logger import Logger
+
+
+
+def find_files(directory, pattern):
+    for root, dirs, files in os.walk(directory):
+        for basename in files:
+            if fnmatch.fnmatch(basename, pattern):
+                filename = os.path.join(root, basename)
+                yield filename
+
+
+l = Logger('Converter')
```

Comment:
`l` is not a nice variable name, both because it is short and because it looks a lot like `1` in many monotype fonts. `p` below isn't nice either, but not as bad. I would appreciate if you would rename them to something better, e.g. `LOGGER` and `PATHS`.